flaxandteal/coral-arches
# Fix: map layer permission issue

Test:

- Create a new user. 
- Give them all the permissions needed to navigate the app. 
- Create an Activity and give it geometry data.
- Add this Activity to the Global Set.
- Add the Global Set to the Global Groups permissions.
- Create a Heritage Asset and give it geometry data.
- Create a Logical Set with this member definition `resource-type-filter=[{"graphid":"076f9381-7b00-11e9-8d6b-80000b44d1d9","name":"Heritage Asset","inverted":false}]`
- Add the Logical Set to the Global Groups permissions.
- Add the new user to the members of the Global Group.
- Run `make manage CMD="apply_sets"`
- Login in as the other user and test if they can see both map layers for Activity and Heritage Asset.
- After remove the Set or Logical Set to test they can't see it any more (re-run apply_sets)
- Remove both and check they also can't see that one as well
